<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageSpeed</class>
 <widget class="QWidget" name="StageSpeed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBk" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#wBk{background-color: rgb(0, 0, 0);}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="wBg" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget#wBg
{border-image: url(:/demoV3_0.png);}</string>
        </property>
        <widget class="QLabel" name="lbTime">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>66</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>12:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcd">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>140</y>
           <width>141</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcd_2">
         <property name="geometry">
          <rect>
           <x>580</x>
           <y>160</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="lbWarning">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>138</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>239</y>
           <width>21</width>
           <height>32</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/c0.png);</string>
          </property>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/c1.png);</string>
          </property>
         </widget>
         <widget class="QWidget" name="page_3">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/c2.png);</string>
          </property>
         </widget>
        </widget>
        <widget class="QStackedWidget" name="stackedWidgetLeft">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-20</y>
           <width>211</width>
           <height>521</height>
          </rect>
         </property>
         <widget class="QWidget" name="page_4">
          <property name="styleSheet">
           <string notr="true">QWidget#page_4{border-image: url(:/bgL_0.png);}</string>
          </property>
         </widget>
         <widget class="QWidget" name="page_5">
          <property name="styleSheet">
           <string notr="true">QWidget#page_5{border-image: url(:/bgL_1.png);}</string>
          </property>
         </widget>
         <widget class="QWidget" name="page_6">
          <property name="styleSheet">
           <string notr="true">QWidget#page_6{border-image: url(:/bgL_2.png);}</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>387</y>
           <width>64</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#widget{background-color: rgb(0,0, 0);}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ItemNum" name="n0" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ItemNum" name="n1" native="true"/>
          </item>
          <item>
           <widget class="ItemNum" name="n2" native="true"/>
          </item>
          <item>
           <widget class="ItemNum" name="n3" native="true"/>
          </item>
          <item>
           <widget class="ItemNum" name="n4" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemNum</class>
   <extends>QWidget</extends>
   <header>ItemNum.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
